src/screens/scanning/VoucherEntryNavigator.tsx

```diff
@@ -0,0 +1,128 @@
+    <TouchableWithoutFeedback onPress={Keyboard.dismiss}>
+      <SafeArea>
+        <StandardHeader width="80%">
+          <TitleText>Add a voucher</TitleText>
+          <TouchableOpacity
+            onPress={() =>
+              handlePreventLeave({
+                hasUnsavedChanges,
+                navigation,
+                dispatch,
+              })
+            }
+          >
+            <Icon name="close" size={24} color={Colors.midBlack} />
+          </TouchableOpacity>
+        </StandardHeader>
+        <View>
+          <NavButtonContainer>
+            <NavButton
+              style={
+                selection === 1 ? styles.selectedBtn : styles.unselectedBtn
+              }
+              onPress={() => {
+                setSelection(1);
+              }}
+            >
+              <NavButtonText
+                style={
+                  selection === 1
+                    ? { color: Colors.magenta }
+                    : { color: Colors.midGray }
+                }
+              >
+                Serial
+              </NavButtonText>
+            </NavButton>
+            <NavButton
+              style={
+                selection === 2 ? styles.selectedBtn : styles.unselectedBtn
+              }
+              onPress={() => {
+                setSelection(2);
+              }}
+            >
+              <NavButtonText
+                style={
+                  selection === 2
+                    ? { color: Colors.magenta }
+                    : { color: Colors.midGray }
+                }
+              >
+                Range
+              </NavButtonText>
+            </NavButton>
+            <NavButton
+              style={
+                selection === 3 ? styles.selectedBtn : styles.unselectedBtn
+              }
+              onPress={() => {
+                setSelection(3);
+              }}
+            >
+              <NavButtonText
+                style={
+                  selection === 3
+                    ? { color: Colors.magenta }
+                    : { color: Colors.midGray }
+                }
+              >
+                Barcode
+              </NavButtonText>
+            </NavButton>
+          </NavButtonContainer>
+        </View>
+        <KeyboardAvoidingView
+          behavior={Platform.OS === 'ios' ? 'padding' : 'height'}
+        >
+          {/* TODO @sauhardjain: Fix typing of navigation attributes */}
+          {(() => {
+            switch (selection) {
+              case 1:
+                return <ManualVoucherScreen navigation={navigation} />;
+              case 2:
+                return <VoucherBatchScreen navigation={navigation} />;
+              case 3:
+                return <ScanningScreen navigation={navigation} />;
+              default:
+                return <ManualVoucherScreen navigation={navigation} />;
+            }
+          })()}
```

Comment:
figure out how to remove red squiggly lines

Reply:
To be fixed in a future PR